autoload.php

```diff
@@ -0,0 +1,32 @@
+<?php
+
+namespace Krizalys\Onedrive;
+
+// The Onedrive SDK autoloader.  You probably shouldn't be using this.  Instead,
+// use a global autoloader, like the Composer autoloader.
+//
+// But if you really don't want to use a global autoloader, do this:
+//
+//     require_once __DIR__ . '/Onedrive/autoload.php'
+
+/**
+ * @internal
+ */
+function autoload($name)
+{
+    // If the name doesn't start with "Krizalys\Onedrive\", then it's not one of our classes.
+    if (substr_compare($name, 'Krizalys\\Onedrive\\', 0, 19) !== 0) return;
+
+    // Take the "Krizalys\\Onedrive\" prefix off.
+    $stem = substr($name, 19);
```

Comment:
In tests I conducted, the name of the class is that way "Krizalys\\Onedrive\\Client", yes, with two bars, so I used 19.